metron-docker/kafkazk/bin/wait-for-it.sh

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env bash
+#   Use this script to test if a given TCP host/port are available
```

Comment:
I could create a pom.xml that essentially replaces the need to manually run metron-docker/install-metron.sh and will instead perform those steps anytime Metron is packaged with Maven.  Will that work?  It should also be forgiving if Docker is not installed and just skip to the next module if it isn't.